hypha/apply/activity/templates/messages/email/base.html

```diff
@@ -1,9 +1,10 @@
 {% block salutation %}Dear {{ user }},{% endblock %}
```

Comment:
I think we should add a `trans` tag for "Dear" as well?

Reply:
Thanks, fix it now.